keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -403,16 +403,24 @@ This section must be completed when targeting beta to a release.
 Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
+It shouldn't impact already running workloads. This is an opt-in feature since 
+users need to explicitly set the minReadySeconds parameter in the StatefulSet spec i.e `.spec.minReadySeconds` field.
+If the feature is disabled the field is preserved. If it was already set in the persisted StatefulSet object, otherwise it is silently dropped.
 
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
+`minReadySeconds` in StatefulSet doesn't get respected and all the `Ready` pods would be shown as `Available`. 
+We consider the feature to be failing if enabling the featuregate and giving
+appropriate value to minReadySeconds doesn't cause `AvailableReplicas` field to be updated 
```

Comment:
how would a cluster-admin know this condition is happening?

Perhaps a cluster-admin would be more interested in watching for failures related to the statefulset controller?  Something like the number of statefulsets without Available replicas growing?

Reply:
I addressed it in the below sentence - https://github.com/kubernetes/enhancements/pull/2824/files#diff-08898e79412d6aad472d55e637efe7f433a78688ac74d94b6d1283b357a1a0a3R421